apollo-compiler/src/main/kotlin/com/apollostack/compiler/FieldTypeSpecBuilder.kt

```diff
@@ -2,18 +2,21 @@ package com.apollostack.compiler
 
 import com.apollostack.compiler.ir.Field
 import com.apollostack.compiler.ir.Fragment
+import com.apollostack.compiler.ir.InlineFragment
 import com.squareup.javapoet.ClassName
 import com.squareup.javapoet.MethodSpec
 import com.squareup.javapoet.TypeSpec
 import javax.lang.model.element.Modifier
 
 class FieldTypeSpecBuilder {
-  fun build(typeName: String, fields: List<Field>, fragments: List<Fragment>): TypeSpec =
+  fun build(typeName: String, fields: List<Field>, fragments: List<Fragment>, inlineFragments: List<InlineFragment>?): TypeSpec =
```

Comment:
sure, that's fine by me, as long as it doens't create horizontal scrollbars in the Github UI. This class right now is not visible in the diff panel for me because it's too wide, but 120 should probably fit